### DISCUSSION TITLE
My selected school has incorrect or incomplete information

### DISCUSSION THREAD

**Discussion by Next-INFINITY** [community/community#120706] [GitHub Education]
### Hi everyone,

Our school [Dalian Minzu University (大连民族大学)] has introduced student-exclusive email addresses, which now end with @stu.dlnu.edu.cn.But our school teachers' email addresses still use the suffix @dlnu.edu.cn.Please assist us in making this change. Thank you.

**Comment by samus-aran** [comment]
Hi @Next-INFINITY  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
